docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[ml-biases]]
+=== Possible biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. People use {ml} in science, healthcare, business, 
```

Comment:
Not mandatory, but I think it's more compelling if we stick with "we" or "you" instead of "people", etc:
```suggestion
across the globe. We use {ml} in science, healthcare, business, 
```